.env.example

```diff
@@ -0,0 +1,3 @@
+# Use backend_user to specify a user to run the backend container as.
+# If none is specified root will be used
+BACKEND_USER=user:group
```

Comment:
Is the "human readable" username okay "dummy:dummy" or does it have to be an UID "42:42"?